presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -935,19 +934,6 @@ public HiveClientConfig setUseParquetColumnNames(boolean useParquetColumnNames)
         return this;
     }
 
-    public boolean isFailOnCorruptedParquetStatistics()
-    {
-        return failOnCorruptedParquetStatistics;
-    }
-
-    @Config("hive.parquet.fail-on-corrupted-statistics")
```

Comment:
do we need to add this to the `DefunctConfig` to avoid breaking the config during upgrade?

Reply:
yes, will add to it